### ISSUE TITLE
Provide supporting links for the download buttons

### ISSUE THREAD

**Issue by gary-rowe** [bitcoin-solutions/multibit-website#169]
Require supporting links for the download buttons providing "How to install" information. Since there is quite a lot of additional info to convey this is best handled by the OS-specific pages.

In JavaScript this can be initiated [with a timed redirect](http://www.willmaster.com/blog/automation/one-link-download-and-redirect.php) after the download has begun. In NoScript this will have to be a static link beneath the buttons.

The "How to install" page should contain:

* link to signature download and instructions on its use
* 32-bit version (if applicable)
* indication of download size in Mb

**Comment by gary-rowe** [comment]
Download buttons now wait for 5s before redirecting the page to the appropriate "how to install" page. These have been checked on FF, Chrome, IE and Safari on Win and OS X. The NoScript version just has a simple link under the buttons.

This issue must be verified to work on an independent machine since the download operation is mission critical.

**Comment by gary-rowe** [comment]
Missed out the redirect code for the primary use case. To verify ensure browser is closed and restarted between page checks. Expected behaviour is:
1. Select browser, enable JavaScript, and navigate to http://localhost:8888
1. Observe buttons greyed out and Accept toolbar in place
1. Click download buttons and observe no activity (wait 10s to be sure)
1. Click Accept and observe transition to enabled buttons
1. Click button and observe immediate start of download (usually with a Run/Save/Open dialog)
1. Wait 5s and observe transition to appropriate "how to install" article
1. Go back to main download page (click icon - might need dedicated link)
1. Refresh page and observe no Accept bar
1. Click download button and see earlier behaviour with transition.
1. Activate NoScript (may need plugin or set JavaScript to off in Tools | Options)
1. Close and restart browser
1. Observe different download buttons with direct links below. 
1. Click download button and observe start of download
1. Click link and observe appropriate "how to install" article
1. Repeat the above for IE, FF, Chrome, Safari, Opera 

Ready for review and close.

**Comment by jim618** [comment]
Safari Test Results
I started up Safari and navigated to http://localhost:8888
The buttons are already enabled and there is no Accept toolbar. (Note I will have run up this site and clicked the Accept button YESTERDAY).
Java script is enabled.

Clicking on each of the Windows, Linux, Mac buttons initiates the downloads and, after a few seconds, transitions to the appropriate Install page.

Checked with NoScript that links appear and navigate to the correct page.


Firefox Test Results.
Start with javascript enabled. Navigate to http://localhost:8888
See the 'Accept bar' ok. Download buttons not enabled.
Click accept. Download buttons enabled. 'Accept bar' disappears.
Click each of the Download buttons in turn. Each correctly offers a 'Save/Run' dialog, downloads the binary and transitions to the install page ok.

With javascript disabled the simple 'how to install' links all appear and work correctly

Chrome Test Results
Open Chrome and navigate to http://localhost:8888
See Download buttons disabled and 'Accept bar'.
Click accept. Alert bar disappears and Download buttons enabled.
Clicking each of the Download button in turn downloads the correct binary and, after a few seconds, navigates to the correct install page

Internet Explorer Test Results
I am testing only on a Mac so haven't tested IE.

Opera Test Results.
Open Opera and navigate to localhost:8888
See disabled Download buttons and Accept bar.
Click Accept. Accept bar disappears and Download buttons enable.
Click on each download button in turn. Each downloads the correct binary and traverses to Install page ok.
Turn off Javascript. See the Install links which work ok. Check the Download buttons download the cirrect binary.

tl;dr Other than (most likely a cache issue) the Accept/Download works fine on Safari/Chrome/Firefox/Opera. IE not checked.

**Comment by gary-rowe** [comment]
OK. That's a good set of tests. I've verified it against IE 11 on Windows 8 and the JavaScript is absolutely vanilla. 

Closing.
